<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uploaddialog</class>
 <widget class="QWidget" name="uploaddialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Upload</string>
  </property>
  <widget class="QTableWidget" name="uploadTableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>611</width>
     <height>91</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>fileName</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>size</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>processBar</string>
    </property>
   </column>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignRight|AlignVCenter</set>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>upload</string>
   </property>
  </widget>
  <widget class="QLabel" name="clientStatusLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
